andela/vale-ah-backend
# #164139776 Create comment

**What does this PR do?**
This PR creates new comments to a recipe.

**Description of Task to be completed?**
- write unit test
- defined the comment schema
- implement create comment

**How should this be manually tested?**
- clone this repository
- run `npm install`
- `npm run devstart`
- On postman:
  * generate an existing user's token using the endpoint, 'localhost:{PORT}/user/login. Ensure the email and password of the user is provided.
  * create a recipe using this endpoint, 'localhost:{PORT}/api/recipes/:slug/comments

**Any background context you want to provide?**
- comments belongsTo a recipe
- comments hasOne user

**What are the relevant pivotal tracker stories?**
#164139776

**Screenshots (if appropriate)**

[#164139774]
